index.js

```diff
@@ -46,6 +47,7 @@ module.exports = async (input = 'patch', options) => {
 	const pkg = util.readPkg();
 	const pkgManager = options.yarn === true ? 'yarn' : 'npm';
 	const pkgManagerName = options.yarn === true ? 'Yarn' : 'npm';
+	const hasLockFile = fs.existsSync('package-lock.json') || fs.existsSync('npm-shrinkwrap.json');
```

Comment:
I just realized, you have to make the paths here relative to the package root, as it should be possible to run `np` in a sub-folder.

You can get the root path with https://github.com/sindresorhus/pkg-dir